packages/expo-video-thumbnails/ios/EXVideoThumbnails.podspec

```
@@ -0,0 +1,20 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'EXVideoThumbnails'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.platform       = :ios, '10.0'
+  s.source         = { git: 'https://github.com/expo/expo.git' }
+  s.source_files   = 'EXVideoThumbnails/**/*.{h,m}'
+  s.preserve_paths = 'EXVideoThumbnails/**/*.{h,m}'
+  s.requires_arc   = true
+
+  s.dependency 'UMCore'
```

```suggestion
  s.dependency 'UMCore'
  s.dependency 'UMFileSystemInterface'
```

you'll have to rerun `pod install` to update other files too 😞